lib/Installation/Partitioner/LibstorageNG/v4/_3/ExpertPartitionerPage.pm

```diff
@@ -0,0 +1,98 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The class introduces all accessing methods for Expert Partitioner
+# Page.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::Partitioner::LibstorageNG::v4::_3::ExpertPartitionerPage;
+use strict;
+use warnings;
+use testapi;
+use parent 'Installation::Partitioner::LibstorageNG::ExpertPartitionerPage';
+
+use YuiRestClient;
+
+sub new {
```

Comment:
probably you already considered, but is it possible to have this in a base class?

Reply:
Sure, we can do it next. There are many things missing or which can be improved, so let's consider those for the future.
I also got an idea that we should have dialog class, as app consists of those, so that we can group widgets in more understandable way.